.travis.yml

```diff
@@ -1,8 +1,16 @@
 language: python
 python:
   - "3.7"
+services:
+  - "docker"
 
 install:
   - "pip install -r requirements.txt"
+  - "curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.35.3/install.sh | bash"
+  - "nvm install"
+  - "nvm use"
+  - "npm install -g gulp"
```

Comment:
This was a naive approach to handling gulp. In reality we should have an npm script that runs lint. Not an issue that needs to be fixed, but should be going forward